src/FrameMapper.cpp

```diff
@@ -263,8 +273,9 @@ void FrameMapper::Init()
 
 			while (remaining_samples > 0)
 			{
-				// get original samples
-				int original_samples = Frame::GetSamplesPerFrame(AdjustFrameNumber(end_samples_frame), original, reader->info.sample_rate, reader->info.channels) - end_samples_position;
+				// Get original samples (with NO framerate adjustments)
+				// This is the original reader's frame numbers
+				int original_samples = Frame::GetSamplesPerFrame(end_samples_frame, original, reader->info.sample_rate, reader->info.channels) - end_samples_position;
```

Comment:
Removing `AdjustFrameNumber()` method from original samples calculation, since this access the original reader, and is always in original frame #'s... never mapped to the Timeline offset.

Reply:
Great! Does this also fix the export issue? When we were getting deformed waveforms when exporting to different FPS?

Reply:
Probably! But I would love to test that again. The new unit test had hundreds of thousands of failures without this fix. And zero failures with it.  But I didn't notice the failures until the Timeline merged the audio, and some frames had the wrong # of samples, and thus introduced some high frequency pops.